Formula/pypy.rb

```diff
@@ -78,42 +80,27 @@ def install
     python = buildpath/"bootstrap/bin/pypy"
 
     cd "pypy/goal" do
-      system python, buildpath/"rpython/bin/rpython",
-             "-Ojit", "--shared", "--cc", ENV.cc, "--verbose",
-             "--make-jobs", ENV.make_jobs, "targetpypystandalone.py"
-
-      with_env(PYTHONPATH: buildpath) do
-        system "./pypy-c", buildpath/"lib_pypy/pypy_tools/build_cffi_imports.py"
-      end
+      system python, "../../rpython/bin/rpython", "--opt", "jit",
+                                                  "--cc", ENV.cc,
+                                                  "--make-jobs", ENV.make_jobs,
+                                                  "--shared",
+                                                  "--verbose"
     end
 
+    system python, "pypy/tool/release/package.py", "--archive-name", "pypy",
+                                                   "--targetdir", ".",
+                                                   "--no-embedded-dependencies",
+                                                   "--no-keep-debug",
+                                                   "--no-make-portable"
```

Comment:
Just aligned these closer to upstream steps: https://doc.pypy.org/en/latest/build.html#run-the-translation https://doc.pypy.org/en/latest/build.html#packaging-preparing-for-installation

Removed `build_cffi_imports.py` as `package.py [--rebuild-extensions]` is the default on macOS, Linux x86_64, and Linux aarch64.